cypress/integration/attributes.js

```diff
@@ -0,0 +1,65 @@
+describe('Attributes test', () => {
+    const AttributeList = ["Strength","Dexterity", "Stamina", "Charisma", "Manipulation", "Composure", "Intelligence", "Wits", "Resolve"]
+    const lvl3Attrs = ["Dexterity", "Stamina", "Charisma"]
+    const lvl2Attrs = ["Manipulation", "Composure", "Intelligence", "Wits"]
+
+    it('All Attributes has initial one dot', () => {
+        cy.visit('http://localhost:3000')        
```

Comment:
posunut von z testu

Reply:
ked to zkusim je tam chyba - cannot call  cy.visit() outside of a running test (have to be inside it)